index.js

```diff
@@ -10,8 +10,12 @@ module.exports = {
 
       this._pretenderPath = resolve.sync('pretender');
       this._pretenderDir = path.dirname(this._pretenderPath);
-      this._routeRecognizerPath = resolve.sync('route-recognizer', { basedir: this._pretenderDir });
-      this._fakeRequestPath = resolve.sync('fake-xml-http-request', { basedir: this._pretenderDir });
+      const resolveOptions = {
+        basedir: this._pretenderDir,
+        preserveSymlinks: false,
```

Comment:
Shouldn't this inherit the value (as opposed to hard coding it)?

Like when launched these ways:

* `node --preserve-symlinks ./node_modules/.bin/ember b`
* `NODE_PRESERVE_SYMLINKS=1 ember b`

Reply:
you want me to add `process.env.NODE_PRESERVE_SYMLINKS !== '1' && process.argv.indexOf('--preserve-symlinks') === -1`?

Reply:
Hehe, well, I'd prefer `resolve` just do the right thing 😝

Reply:
But since _that_ won't happen, I suppose that guard is the best we can do...